prime-angular-frontend/src/app/modules/health-auth-site-reg/shared/models/health-authority-admin-site-list.model.ts

```diff
@@ -34,7 +34,8 @@ export class HealthAuthoritySiteAdminList extends AbstractBaseHealthAuthoritySit
     public isNew: boolean,
     public readonly authorizedUserName: string,
     public readonly authorizedUserEmail: string,
-    public readonly adjudicatorIdir: string
+    public readonly adjudicatorIdir: string,
+    public duplicatePecSiteCount: number
```

Comment:
This seems like it needs clarification ... It is the number of OTHER sites that have the same Site ID as this row?

Reply:
Added comment